discord/interaction_response.go

```diff
@@ -5,7 +5,9 @@ type InteractionResponseType int
 
 // Constants for the InteractionResponseType(s)
 const (
-	InteractionResponseTypePong InteractionResponseType = iota + 1
+	// This is stricly internal and will never be sent to discord. It is used to indicate that the HTTP response should be 202 Accepted
+	InteractionResponseTypeAcknowledge InteractionResponseType = iota
+	InteractionResponseTypePong        InteractionResponseType = iota
```

Comment:
`iota` is only needed for the first entry in a `const` block
that being said I think i would prefer a magic value of `-1` instead of `0`.

Your comment should also be prefixed with `InteractionResponseTypeAcknowledge`